stratum/hal/lib/tdi/tdi_sde_helpers.cc

```diff
@@ -392,7 +392,7 @@ ::util::Status SetFieldBool(::tdi::TableData* table_data,
 
 ::util::Status GetAllEntries(
     std::shared_ptr<::tdi::Session> tdi_session,
-    ::tdi::Target tdi_dev_target, const ::tdi::Table* table,
+    ::tdi::Target &tdi_dev_target, const ::tdi::Table* table,
```

Comment:
Do sizeGet and usageGet API's on line 410 and 414 accepts tdi_dev_target by reference?